doc/modules/classes.rst

```diff
@@ -143,43 +143,58 @@ Classes
    covariance.oas
    covariance.graph_lasso
 
+.. _model_selection_ref:
 
-.. _cross_validation_ref:
-
-:mod:`sklearn.cross_validation`: Cross Validation
-=================================================
+:mod:`sklearn.model_selection`: Model Selection
+===============================================
 
-.. automodule:: sklearn.cross_validation
+.. automodule:: sklearn.model_selection
    :no-members:
    :no-inherited-members:
 
-**User guide:** See the :ref:`cross_validation` section for further details.
+**User guide:** See the :ref:`cross_validation`, :ref:`grid_search` and
+:ref:`learning_curve` sections for further details.
+
 
+Classes
+-------
 .. currentmodule:: sklearn
 
 .. autosummary::
    :toctree: generated/
    :template: class.rst
 
-   cross_validation.KFold
-   cross_validation.LeaveOneLabelOut
-   cross_validation.LeaveOneOut
-   cross_validation.LeavePLabelOut
-   cross_validation.LeavePOut
-   cross_validation.PredefinedSplit
-   cross_validation.StratifiedKFold
-   cross_validation.ShuffleSplit
-   cross_validation.StratifiedShuffleSplit
+   model_selection.KFold
+   model_selection.LeaveOneLabelOut
+   model_selection.LeaveOneOut
+   model_selection.LeavePLabelOut
+   model_selection.LeavePOut
+   model_selection.PredefinedSplit
+   model_selection.StratifiedKFold
+   model_selection.ShuffleSplit
+   model_selection.StratifiedShuffleSplit
+   model_selection.GridSearchCV
+   model_selection.ParameterGrid
+   model_selection.ParameterSampler
+   model_selection.RandomizedSearchCV
+
+
+Functions
+---------
+.. currentmodule:: sklearn
 
 .. autosummary::
    :toctree: generated/
    :template: function.rst
 
-   cross_validation.train_test_split
-   cross_validation.cross_val_score
-   cross_validation.cross_val_predict
-   cross_validation.permutation_test_score
-   cross_validation.check_cv
+   model_selection.train_test_split
+   model_selection.cross_val_score
+   model_selection.cross_val_predict
+   model_selection.permutation_test_score
+   model_selection.check_cv
```

Comment:
maybe a stupid question, but why is ``check_cv`` public? Input from @vene @gaelvaroquaux @larsmans @jnothman welcome.